lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -10,13 +10,15 @@ class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Vm < MiqAeServiceVmC
     expose :validate_resize_confirm
     expose :validate_resize_revert
 
+    expose :associate_floating_ip
+    expose :disassociate_floating_ip
```

Comment:
It might be needed add ```:override_return => nil``` after expose method, see conversation with GregM at https://github.com/ManageIQ/manageiq/pull/7922